src/main/resources/application.properties

```diff
@@ -5,6 +5,7 @@ spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.SQLServerDialect
 spring.jpa.hibernate.ddl-auto = update
 spring.jpa.open-in-view=false
 spring.jpa.properties.hibernate.enable_lazy_load_no_trans=true
+spring.jpa.properties.hibernate.legacy_limit_handler=true
```

Comment:
to słowo legacy nie jest zbyt zachęcające XD

Reply:
Bez tego nie chcą działac maxResults i firstResult z hibernejta. SQL Server ma jakieś problemy. wgl w przyszlosci chyba go wymienimy na MySQLa czy cos.